exercises/concept/annalyns-infiltration/.meta/exemplar.cairo

```diff
@@ -0,0 +1,17 @@
+pub fn can_fast_attack(knight_awake: bool) -> bool {
+    return !knight_awake;
```

Comment:
```suggestion
    !knight_awake
```
nit: returning a value _without_ using `return` is a more "Cairo" way, update the other returns as well